system-test/bigquery.js

```diff
@@ -505,6 +505,185 @@ describe('BigQuery', function() {
         }
       );
     });
+
+    describe('location', function() {
+      var dataset = bigquery.dataset(generateName('dataset'));
+      var table = dataset.table(generateName('table'));
+      var job;
+
+      var LOCATION = 'asia-northeast1';
+      var QUERY = `SELECT * FROM \`${table.id}\``;
+      var SCHEMA = require('./data/schema.json');
+      var TEST_DATA_FILE = require.resolve('./data/location-test-data.json');
+
+      before(function() {
+        // create a dataset in a certain location will cascade the location
+        // to any jobs created through it
+        return dataset.create({location: LOCATION}).then(function() {
+          return table.create({schema: SCHEMA});
+        });
+      });
+
+      it('should create a load job', function() {
+        return table.createLoadJob(TEST_DATA_FILE).then(function(data) {
```

Comment:
This should be done in the before so that the other tests don't depend on this test being run and succeeding. This test can either be skipped, or it can be something like a check on the metadata to be sure the name/location/metadata was set properly.

Reply:
Done!